comps/comps-foreman-rhel7.xml

```diff
@@ -33,6 +33,7 @@
       <packagereq type="default">foreman-service</packagereq>
       <packagereq type="default">foreman-telemetry</packagereq>
       <packagereq type="default">foreman-vmware</packagereq>
+      <packagereq type="default">nodejs-graphql</packagereq>
```

Comment:
I'd expect this to be tfm-nodejs-graphql for EL7?

(and the next line too, but that's not part of this pr)